lib/ansible_tower_client/group.rb

```diff
@@ -1,14 +1,6 @@
 module AnsibleTowerClient
-  class Group
+  class Group < BaseModel
     extend CollectionMethods
-    include InstanceMethods
-
-    attr_reader :inventory_id
-
-    def initialize(raw_body)
-      @inventory_id = raw_body["inventory"]
```

Comment:
You can define your own method for particular model, for example
```
def inventory_id
  __getobj__['inventory']
end
```
The auto generated `inventory` method still exists and returns the same thing.